.travis.yml

```
@@ -13,7 +13,8 @@ env:
     - ISTANBUL_COVERAGE: yes
 
 script:
- - yarn bootstrap --env webpack-4
+ - nvm install 8.10.0
```

Is it necessary to use a deprecated version and support of highly outdated webpack versions?
```suggestion
 - nvm install 8.17.0
```